Sources/Identity/CustomerInfo.swift

```diff
@@ -19,7 +19,7 @@ import Foundation
  A container for the most recent customer info returned from `Purchases`.
  These objects are non-mutable and do not update automatically.
  */
-@objc(RCCustomerInfo) public class CustomerInfo: NSObject {
+@objc(RCCustomerInfo) public final class CustomerInfo: NSObject {
```

Comment:
> 🤔 is this necessary?

It is to be able to use `Self` in the `HTTPResponseBody` implementation. I could make that explicitly use `CustomerInfo`, but.

> I'm wondering if this might make it trickier for customers to use our classes for testing if they're doing inherit+replace

None of our classes are `open`. Since [Swift 3, Swift classes are sealed (non-open) by default](https://forums.swift.org/t/proposal-sealed-classes-by-default/3164). So this being `final` only affects the SDK itself.